src/theory/quantifiers/sygus/ce_guided_single_inv_sol.cpp

```diff
@@ -702,7 +702,7 @@ Node CegConjectureSingleInvSol::reconstructSolution(Node sol,
             // if it is not already reconstructed
             if( d_reconstruct.find( itt->second )==d_reconstruct.end() ){
               Trace("csi-rcons") << "...reconstructed " << ns << " for term " << nr << std::endl;
-              bool do_check = true;//getPathToRoot( itt->second );
+              constexpr bool do_check = true;//getPathToRoot( itt->second );
```

Comment:
I would be fine if you wanted to eliminate this variable and simplify the code accordingly.

Reply:
Done.